### PR TITLE
test(component): enable fail fast

### PR DESCRIPTION
**Pull request by adamdehaven**
# Summary

Add the `cypress-fail-fast` plugin.

Note: this will not stop execution of all spec files, but will skip all remaining tests in a spec if one test in the spec fails.

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Review by portikM — DISMISSED** [review]

File: src/components/KCatalog/KCatalog.cy.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ const largeDataSet = [
  */
 
 describe('KCatalog', () => {
-  function getItems(count) {
+  function getItems(count: number) {
```
Comment (ValeryG) [inline src/components/KCatalog/KCatalog.cy.ts:58]: I wish it would fix test instability :)

**Review by ValeryG — APPROVED** [review]

File: src/components/KCatalog/KCatalog.cy.ts (hunk 1 above)
Comment (adamdehaven) [reply src/components/KCatalog/KCatalog.cy.ts:58]: I'm going to open a ticket -- we need to figure out why `KCatalog.cy.ts` fails

**Review by portikM — APPROVED** [review]

**Review by kaiarrowood — APPROVED** [review]

File: src/components/KCatalog/KCatalog.cy.ts (hunk 1 above)
Comment (kaiarrowood) [reply src/components/KCatalog/KCatalog.cy.ts:58]: If you want, assign to me, I might have some cycles to do a deep dive/refactor

File: src/components/KDropdownMenu/KDropdownMenu.cy.ts [hunk 2]
```diff
@@ -145,7 +145,7 @@ describe('KDropdownMenu', () => {
     cy.get('.k-dropdown-popover').should('contain.html', itemSlotContent)
   })
 
-  it('correctly renders dividers on all item types', () => {
+  it.only('correctly renders dividers on all item types', () => {
```
Comment (kaiarrowood) [inline src/components/KDropdownMenu/KDropdownMenu.cy.ts:148]:
```suggestion
  it('correctly renders dividers on all item types', () => {
```

**Review by kaiarrowood — APPROVED** [review]
LGTM! 🚀

**Pull request merged**